src/OperationalInsights/OperationalInsights/Tables/UpdateAzureOperationalInsightsTableCommand.cs

```diff
@@ -17,8 +17,8 @@
 
 namespace Microsoft.Azure.Commands.OperationalInsights.Tables
 {
-    [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsTable", SupportsShouldProcess = true, DefaultParameterSetName = ByWorkspaceName), OutputType(typeof(PSWorkspace))]
-    public class SetAzureOperationalInsightsTableCommand : OperationalInsightsBaseCmdlet
```

Comment:
Removing cmdlet dill cause breaking change. Please add `Set-AzOperationalInsightsTable` as an alias of `Update-AzOperationalInsightsTable`.
And please add a breaking change warning message here.

Reply:
renamed cmdlet to "Set" in order to prevent breaking change